docs/usage/cvc_guide.rst

```diff
@@ -75,18 +75,28 @@ by editing JSONL, to ensure that you do not add any additional linebreaks.
 .. note::
     - If your dataset does not have JSONL files, committing it via the Conservator UI will generate those files, and they will be available the next time you clone or pull the dataset.
     - Any changes pushed to a dataset via ``cvc`` will be reflected in both ``index.json`` and the relevant JSONL file after the dataset has been pulled.
-    - If you try to commit and push changes to both ``index.json`` and one of the JSONL files at the same time, it will fail. You can only make changes to ``index.json`` or one of the JSONL files, but not to both at the same time.
+    - If you try to commit and push changes to both ``index.json`` and one of the JSONL files at the same time, it will fail. You cannot push changes to ``index.json`` and one of the JSONL files at the same time.
 
 
 Adding Frames
 ^^^^^^^^^^^^^
 
-You can stage new frames to be uploaded to a dataset using the ``add`` command::
+You can stage new frames to be uploaded to a dataset using the ``stage-image`` command::
+
+    $ cvc stage-image path/to/some/file.jpg ../some/other/path.jpg etc.jpg
+    $ cvc stage-image ./someimages/*.jpg
```

Comment:
This is some good documentation around the feature. One thing that's a little unclear to me is if you should ever stage your images from the `data/` directory. Would you do that with the `--skip-copy` flag or is it completely discouraged? Depending on the recommended behavior consider adding a warning with additional usage hints if the user tries to stage images directly from the current dataset's `data/`

Reply:
Honestly, I am not sure why the `--skip-copy` flag exists. When cvc uploads an image, it also generates an id for the new frame, and copies the image to the `data/` folder with a new name based on that new id, the dataset id, and other variables, so if you staged an image from `data/`, it would remain there, but would not affect anything else. if you use the `--skip-copy` flag, obviously the image would not get copied into `data/`, but I cannot think of a use-case where you would want to do that.

Reply:
that makes sense, thanks Neal! Maybe we'll re-evaluate `--skip-copy` down the road, but it does not seem like it will hurt anything.